src/Menu.java

```diff
@@ -0,0 +1,156 @@
+import org.json.simple.JSONArray;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Menu {
+
+
+    public static void main(String[] args){
+
+        try {
+            int userInput;
+            do {
+                Utils.clearScreen();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
I recommend using `Keyboard.nextInt()` instead of creating a `BufferedReader` object. It avoids creating multiple objects and mixing between `Scanner` and `BufferedReader` for stdin.